examples/multiplatform-example/shared/ci.properties

```diff
@@ -0,0 +1,2 @@
+api_key="ci_api_key"
+number_key=12
```

Comment:
Shouldn't this value be different in order to ensure the value we get is not from the ``debug.properties`` or any other file?

Reply:
if you are referring to `number_key` and the difference between `debug.properties` and `ci.properties`, those files are from different projects example, those values are different to the default one in `gradle.properties` on `multiplatform-example`.

In the project `kotlin-example` there are `ci.properties` and `debug.properites` with different variables.